eng/targets/Helix.props

```diff
@@ -22,7 +22,6 @@
     <HelixUseArchive>false</HelixUseArchive>
     <LoggingTestingDisableFileLogging Condition="'$(IsHelixJob)' == 'true'">false</LoggingTestingDisableFileLogging>
     <NodeVersion>10.15.3</NodeVersion>
-    <AppRuntimeVersion>5.0.0-ci</AppRuntimeVersion>
```

Comment:
I also wonder if this hard-code version was causing problems when using `eng/scripts/RunHelix.ps1` locally, where the version would be `5.0.0-dev` even now❔

Reply:
@HaoK what do you thunk❔

Reply:
Yes for sure, that's what we use for the directory we install to, if you were seeing weirdness with the bits not installed correctly this would be why